### PR TITLE
[docs] Update Credentials 

### PR DESCRIPTION
**Pull request by theBeginner86**
**Notes for Reviewers**

1. Adds Credentials sections
2. Moves summary section of Environments and Workspaces to last section.

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit bbd968950384c4fbf249b72252eba6bdeef13f60 at: https://654226de48e7e6057e0369d8--meshery-docs-preview.netlify.app

**Comment by theBeginner86** [comment]
Is this incomplete/ambiguous? Should I improve this?

File: docs/pages/concepts/logical/credentials.md [hunk 1]
```diff
@@ -7,4 +7,40 @@ abstract: Meshery uses one or more Credentials when authenticating to a managed
 language: en
 list: include
 ---
-Meshery uses one or more Credentials when authenticating to a managed or unmanaged Connection.
\ No newline at end of file
+Meshery uses one or more Credentials when authenticating to a managed or unmanaged Connection.
```
Comment (leecalcote) [inline docs/pages/concepts/logical/credentials.md:10]: Technically, this is zero or more, however, that phrasing is a little difficult to digest, so, if you can include the concept of Meshery potentially not using a credential... great.

File: docs/pages/concepts/logical/credentials.md [hunk 2]
```diff
@@ -7,4 +7,40 @@ abstract: Meshery uses one or more Credentials when authenticating to a managed
...
+Credentials are pivotal in Meshery as they serve as the authentication mechanism for connections registered with the Meshery server. They ensure secure interactions between Meshery and the various services and systems Meshery is designed to work with. <br>
```
Comment (leecalcote) [inline docs/pages/concepts/logical/credentials.md:11]: Extraneous, and perhaps, inaccurate to the extent that Meshery doesn't even use credentials sometimes.

File: docs/pages/concepts/logical/credentials.md [hunk 3]
```diff
@@ -7,4 +7,40 @@ abstract: Meshery uses one or more Credentials when authenticating to a managed
...
+For more info, see [Meshery Integrations](/integrations/).
+
+
+## Key Features
```
Comment (leecalcote) [inline docs/pages/concepts/logical/credentials.md:15]: Not ideal heading. Let's focus on the value offered or the task at hand instead.

File: docs/pages/concepts/logical/credentials.md [hunk 4]
```diff
@@ -7,4 +7,40 @@ abstract: Meshery uses one or more Credentials when authenticating to a managed
...
+
+- **Granular Access Control**
+Meshery provides fine-grained access control through credentials. You can specify which credentials are associated with particular connections and workspace, giving you control over who can access and manage specific resources.
+<br>
+For more information, see "[Workspaces](/concepts/workspaces)".
+
+
+## Examples
+
+### Kubernetes Connections
```
Comment (leecalcote) [inline docs/pages/concepts/logical/credentials.md:40]: We need an entire page dedicated to Kubernetes (platform) connection management, @MUzairS15

**Comment by leecalcote** [comment]
merge conflict

**Comment by leecalcote** [comment]
Merge conflict...

**Pull request closed without merging**